change/@ni-jasmine-parameterized-7779e285-bc01-469c-8e6d-ae0b039d8273.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's add this package to the root repo README under "And some additional utility packages:"